specs/IsPinchZoomEnabled.md

```diff
@@ -0,0 +1,86 @@
+# Background
+
+There are two types of zoom in Chromium: Browser Zoom and Pinch-Zoom:
+- Browser zoom, referred to as “Page Zoom” or “Zoom” more generally, is what we get by using Ctrl + +(plus) or – (minus) or Ctrl + Mousewheel. This changes the size of a CSS pixel relative to a device independent pixel and so it will cause page layout to change. End developers can programmatically change browser Zoom properties including [IsZoomControlEnabled](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2settings?view=webview2-1.0.774.44#get_iszoomcontrolenabled) and [ZoomFactor](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2controller?view=webview2-1.0.774.44#get_zoomfactor). Setting ZoomFactor property causes the layout to change and enables scroll bar which lets users interact with the zoomed in content using mouse.
+- Pinch-zoom, referred to as “Page Scale” zoom, is performed as a post-rendering step, it changes the page scale factor property and scales the surface the web page is rendered onto when user perfoms a pinch zooming action. It does not change the layout but rather changes the viewport and clips the web content, the content outside of the viewport isn't visible onscreen and users can't reach this content using mouse.
+
+Currently, the first type of zoom control is supported in WebView2 and modifying it has no effect on page scale zoom. 
+
+In response to customer requests to be able to change the current functionality of page scale zoom in WebView2, the WebView2 team has introduced this Pinch Zoom API which allows end developers to disable or enable page scale zoom control via a setting.
+
+In this document we describe the new setting. We'd appreciate your feedback.
+
+
+# Description
+The default value for `IsPinchZoomEnabled` is `true`.
+When this setting is set to `false`, it disables the ability of the end users to use pinching motions on touch input enabled devices to scale the web content in the WebView2 and users cannot pinch zoom.
+
```

Comment:
Does this property take effect immediately? Or does it not take effect until the next navigation (like most other settings).

If it takes effect immediately, then presumably setting it to `false` when the user has already pinch-zoomed means that the pinch-zoom is instantly reset to 1.0.

Reply:
Please update docs to note when it takes effect. Also, update docs to explicitly note if disabling IsPinchZoomEnabled changes the page scale or just disables the end user's ability to use pinch motions